src/components/vpncard.js

```diff
@@ -250,3 +250,117 @@ export class VPNCard extends LitElement {
   }
 }
 customElements.define("vpn-card", VPNCard);
+
+/**
+ * This is a "Dummy Card" that shows a loading animation.
+ */
+export class VPNCardPlaceHolder extends LitElement {
+  render() {
+    return html`
+      <main class="box">
+        <div style="filter:grayscale();">
```

Comment:
Curiosity/nit: Is there a reason we have these inline instead of in the CSS?

Reply:
Lazyness :)